Workbooks/Workloads/Update monitoring vm config/Update monitoring vm config.workbook

```diff
@@ -280,7 +288,7 @@
     {
       "type": 1,
       "content": {
-        "json": "#### Enable access to Azure Key Vaults (Optional)\nIf your connection strings reference secrets from Key Vaults, you need to provide access to it from the virtual machine via a Managed Service Identity."
+        "json": "#### Enable access to Azure Key Vaults\nPlease provide access to the virtual machine from the Key Vaults referenced in the connection strings via Managed Service Identity."
```

Comment:
`#### Enable access to Azure Key Vault\nUse Key Vault access policy to provide the Get Secret permission for all key vaults referenced in configuration. Permission must be provided to the Managed Service Identity of the monitoring virtual machine.`